card/mod/05_standard/set/type/search_type.rb

```diff
@@ -92,9 +92,13 @@ def search_results args={}
       if search_vars[:error]
         search_vars[:error]
       else
-        raw_results = card.item_cards search_params
-        is_count = search_vars[:query][:return] =='count'
-        is_count ? raw_results.to_i : raw_results
+        begin
+          raw_results = card.item_cards search_params
+          is_count = search_vars[:query][:return] =='count'
```

Comment:
Surrounding space missing for operator '=='.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.